src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -18,7 +18,7 @@ private MobileAdsDataProvider() {
     return new Object[][]{
         {
             "adtest", "SyntheticTests/Slots/leaderboard+prefooter",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "adtest" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)